packages/component-library/src/BaseMap/BaseMap.js

```diff
@@ -182,18 +188,32 @@ class BaseMap extends Component {
         ? CIVIC_DARK
         : CIVIC_LIGHT;
 
+    const animationProps = !animate
+      ? {}
+      : {
+          transitionDuration: 1000,
```

Comment:
Ideally we should be able to modify the duration, I think.

Reply:
Good call.